config/locales/en/providers.yml

```diff
@@ -270,6 +270,7 @@ en:
         h1-heading: Latest incident details
     declarations:
       show:
+        h1-heading: Declaration
```

Comment:
This appears to be a duplicate of the line below

Reply:
Thanks, will remove